docs/docs/tuning-your-model.mdx

```diff
@@ -364,3 +364,95 @@ You may want to limit the absolute amount of GPU memory that can be used by a Ra
 
 For example, say you have two visible GPUs(`GPU:0` and `GPU:1`) and you want to allocate 1024 MB from the first GPU
 and 2048 MB from the second GPU. You can do this by setting the environment variable `TF_GPU_MEMORY_ALLOC` to `"0:1024, 1:2048"`.
+
+
+## Accessing Diagnostic Data
+
+To gain a better understanding of what your models do, you can access intermediate results of the prediction process.
+To do this, you need to access the `diagnostic_data` field of the [Message](./reference/rasa/shared/nlu/training_data/message.md#message-objects)
+and [Prediction](./reference/rasa/core/policies/policy.md#policyprediction-objects) objects, which contain
+information about attention weights and other intermediate results of the inference computation.
+You can use this information for debugging and fine-tuning, e.g. with [RasaLit](https://github.com/RasaHQ/rasalit).
+
+After you've [trained a model](./reference/rasa/command-line-interface.md#rasa-train), you can access diagnostic data for DIET like this 
+(please define the `YOUR_RASA_MODEL_DIRECTORY` and `YOUR_RASA_MODEL_NAME` constants):
```

Comment:
I'm not sure if full length python code is best for this, if anything underneath changes it will not work and we don't currently have CI that runs these files to make sure.

Reply:
Same goes for in the changelog, its also a very long changelog entry...

Reply:
I think it's ok if the changelog is specific to the version where the change occurred. I can modify the example in the docs, though. @wochinge What do you think?

Reply:
@erohmensing I've shortened the code in the documentation. I think earlier Tobi had asked me to add the full examples to the changelog (but I can't find that anymore, so maybe I misremember). So I don't know what to do.

Reply:
What about having a GitHub gist for the changelog? Shortens the changelog but the information stays there. + I like the shortened example in the code. We have all sorts of examples in the code which aren't tested so one more for this shouldn't be a problem in my opinion.

Reply:
Ok, I moved the example to a Ghist.